pdr_backend/subgraph/subgraph_payout.py

```diff
@@ -135,7 +135,6 @@ def fetch_payouts(
     )
 
     try:
-        logger.info("Querying subgraph... %s", query)
         result = query_subgraph(
```

Comment:
why were these logging lines removed?

Reply:
Because they are horribly length and noisy.
I'm the most power user of the lake and I can't follow what's going on.

What's the point of logging hundreds of queries that all have the same text?